test_outputs/query_execution/trait_method_marked_deprecated.snap

```diff
@@ -3,30 +3,6 @@ source: src/query.rs
 expression: "&query_execution_results"
 ---
 {
-  "./test_crates/trait_marked_deprecated/": [
-    {
-      "method_name": String("method"),
-      "name": String("TraitToDeprecatedTrait"),
-      "path": List([
-        String("trait_marked_deprecated"),
-        String("TraitToDeprecatedTrait"),
-      ]),
-      "span_begin_line": Uint64(5),
-      "span_filename": String("src/lib.rs"),
-      "visibility_limit": String("public"),
-    },
-    {
-      "method_name": String("method"),
-      "name": String("TraitToDeprecatedMessageTrait"),
-      "path": List([
-        String("trait_marked_deprecated"),
-        String("TraitToDeprecatedMessageTrait"),
-      ]),
-      "span_begin_line": Uint64(10),
-      "span_filename": String("src/lib.rs"),
-      "visibility_limit": String("public"),
-    },
-  ],
```

Comment:
Hmm I should've catch it up, would be more careful next time!

Reply:
Learning is the most important thing!